### PR TITLE
Add initial tests for Timeline PointItem

### PR DESCRIPTION
**Pull request by bradh**
I'm looking to add some fancier item types, this is initial work on understanding what is already there and how it works.

### CODE REVIEW

File: test/TestSupport.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            domProps: this.props,
+            emitter: {
+                on: function() {},
+                off: function() {},
+                emit: function() {}
+            },
+            hiddenDates: [],
+            util: {}
+            };
+            body.dom.rollingModeBtn = document.createElement('div');
```
Comment (yotamberk) [inline test/TestSupport.js]: Why did you add a rollingModeBtn?

Comment (bradh) [inline test/TestSupport.js]: I didn't (well, not in this PR). That is existing code from https://github.com/almende/vis/blob/develop/test/TimelineRange.test.js#L24 that is needed for the part of the Range constructor that adds an event listener: https://github.com/almende/vis/blob/develop/lib/timeline/Range.js#L69

File: test/TestSupport.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (mojoaxel) [inline test/TestSupport.js]: The indent in this file is a little off. Can you please change it to something consistent with the other files. Thanks!

Comment (bradh) [inline test/TestSupport.js]: So just two spaces?

Comment (mojoaxel) [inline test/TestSupport.js]: ok

File: test/TestSupport.js (hunk 1 above)
Comment (yotamberk) [inline test/TestSupport.js]: But it seems weird that it's needed for the test without being even used

Comment (bradh) [inline test/TestSupport.js]: It is needed to create a Range object, which I use for the "isVisible" checks.
